common/lib/xmodule/xmodule/course_module.py

```diff
@@ -741,6 +741,34 @@ class CourseFields(object):
         scope=Scope.settings,
     )
 
+    # Edraak (progs): linking a course with a specialization
+    specialization_title_en = String(
+        display_name="Specialization Title",
```

Comment:
Rename the display_name to `Specialisation Title (English)` and `Specialisation Title (Arabic)`